src/Files.App/UserControls/Pane/ShelfPane.xaml

```diff
@@ -48,6 +71,30 @@
 
 		</StackPanel>
 
+		<!--  Empty Shelf Content  -->
+		<StackPanel
+			x:Name="EmptyShelfContent"
+			Grid.Row="1"
+			Padding="16,0,16,0"
+			VerticalAlignment="Center"
+			Orientation="Vertical"
+			Spacing="16"
+			Visibility="{x:Bind ItemsSource.Count, Converter={StaticResource EmptyListToVisibilityConverter}, Mode=OneWay}">
```

Comment:
Alternatively, `x:Load` could be used (but I doubt it'd make a large difference)

Reply:
I plan to add some VisualStates which will require using `Visibility`.